docs/UserGuide.md

```diff
@@ -61,25 +74,128 @@ Taskify is a **desktop app for students manage their tasks (academics/personal/C
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+</div>
+
+
+
+### Viewing help : `help`
+
+Shows a message explaning how to access the help page.
+
+![help message](images/helpMessage.png)
 
+Format: `help`
 
 
-### Command Action: `keyword(first word of command)`
+### Adding a task : `add`
 
-Command Desription
+Adds a task to Taskify.
 
-Format: `command format`
+Format: `add n/NAME desc/DESCRIPTION a/ADDRESS date/DATE [t/TAG]…`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `example 1`
-* `example 2`
+* `add n/myTask1 desc/1st Task a/John street, block 123, #01-01 date/2020-04-13`
```

Comment:
can you replace lines 1-2 to these
```
---
layout: page
title: User Guide
```
I think this is needed for the website to display UG and DG links and you removed these lines in the previous PR.
Currently, if you check the netlify build for the website, there are no links to the UG and DG